lib/npm.js

```diff
@@ -84,6 +84,12 @@ async function get (name, etag) {
   }
 }
 
+async function post (path, body) {
+  let opts = Object.assign({body}, getRequestOpts())
```

Comment:
I'd prefer to use the const declarations here because these values are never reassigned.